webapp/users.py

```diff
@@ -62,10 +76,10 @@ def get_id(self):
         """This class attrubute holds the user account's ID"""
         return self._id
 
-users = {'admin': User(u'admin'), 'anonymous': AnonUser()}
+users = {'admin': User(u'admin','"admin'), 'anonymous': AnonUser()}
```

Comment:
We can probably get rid of the `AnonUser` class and remove the `'anonymous': AnonUser()` user from this pre-existing users list. That is more of an artifact that I was playing around with when I [half-assed] setup the user system.